CONTRIBUTING.md

```diff
@@ -110,6 +111,17 @@ Use your real name (sorry, no pseudonyms or anonymous contributions.)
 If you set your `user.name` and `user.email` git configs, you can sign your
 commit automatically with `git commit -s`.
 
+### Integration Tests
+
+Our primary means of performing integration testing for libpod is with the
+[Ginkgo](https://github.com/onsi/ginkgo) BDD testing framework. This allows
+us to use native Golang to perform our tests and there is a strong affiliation
+between Ginkgo and the Go test framework.  Adequate test cases are expected to
+be provided with PRs.
+
+For details on how to run the tests for Podman in your test environment, see the
+Integration Tests [README.md](test).
```

Comment:
My [earlier comment][1] wasn't about `README` vs. `README.md` in the link text (I'm fine either way).  I'd just rather avoid a disconnect between the nominal and actual target.  Currently your nominal target is the README file, but the link actually points to the directory.  I'd rather say we're linking the directory if we use the directory parth, or say we're using the README and use the README path.  But because GitHub will display the README content via either URL, this is just my consistency nit and you can leave it unaddressed if you want ;).

[1]: https://github.com/projectatomic/libpod/pull/910#discussion_r193254735

Reply:
Got'cha, touched up as I'd to chase out a rouge space in one file.